api/src/services/authorization.ts

```diff
@@ -498,7 +498,7 @@ export class AuthorizationService {
 
 		const preset = permission.presets ?? {};
 
-		const payloadWithPresets = merge({}, preset, payload);
+		const payloadWithPresets = merge({}, parsePreset(preset, this.accountability, {}), payload);
```

Comment:
Correct, i was just unable to find any content object 'nearby' also don't know how deep it should work, ie. $CURRENT_USER.tenant.is_admin

Reply:
I'll have to look up where the other context is created, IIRC there's a util function that looks at the used variables and resolves the context, but my memory is a little hazy on that

Reply:
Found them. Ran into a circular dependency due to ItemsService being used everywhere :) So ended up refactoring a bit in order to get that to work.

All tests are passing, and validation and presets work now with context of `$CURRENT_USER` and `$CURRENT_ROLE`

I was tempted to add support for `$CURRENT_ITEM` since I see how that might be able to work, but will leave that for a different PR.